models/ctsm/DESCRIPTION

```diff
@@ -1,17 +1,40 @@
 Package: PEcAn.CTSM
 Type: Package
-Title: PEcAn package for integration of the CLM model. Will support CLM v4.5 and 5, and will provide helper functions for PEcAn.FATES 
+Title: PEcAn package for integration of the CTSM models.
 Version: 1.7.0
 Date: 2018-12-03
-Authors@R: c(person("Mike","Dietze"), person("Shawn","Serbin"), person("Andy","Fox"), person("David","LeBauer"))
-Author: Mike Dietze, Shawn Serbin, Andy Fox, David LeBauer
-Maintainer: Mike Dietze <dietze@bu.edu>
+Authors@R: c(person("David", "LeBauer", role = c("aut", "cre"),
+                     email = "dlebauer@email.arizona.edu",
+                     comment = c(ORCID = "0000-0001-7228-053X")),
+             person("Mike", "Dietze", role = c("aut"),
+                     email = "dietze@bu.edu",
+                     comment = c(ORCID = "0000-0002-2324-2518")),
+             person("Rob", "Kooper", role = c("aut"),
+                     email = "kooper@illinois.edu",
+                     comment = c(ORCID = "0000-0002-5781-7287")),
+             person("Shawn", "Serbin", role = c("aut"),
+                     email = "sserbin@bnl.gov",
+                     comment = c(ORCID = "0000-0003-4136-8971")),
+             person("Betsy", "Cowdery", role = c("aut"),
+                     email = "ecowdery@bu.edu",
+                     comment = c(ORCID = "0000-0002-6538-6296")),
+             person("Baily", "Morrison", role = c("aut"),
+                     email = "bmorrison@bnl.gov"),
+             person("Kristina", "Riemer", role = c("aut"),
+                     email = "kristinariemer@email.arizona.edu"),
+             person("Tonantzin", "Tarin", role = c("aut"),
+                     email = "ttarin@udel.edu"),
+             person("Tony", "Gardella", role = c("aut"),
+                     email = "tonygard@bu.edu",
+                     comment = c(ORCID = "0000-0003-4380-3412")))
 Description: The Predictive Ecosystem Analyzer (PEcAn) is a scientific
     workflow management tool that is designed to simplify the management of
     model parameterization, execution, and analysis. The goal of PEcAn is to
     streamline the interaction between data and models, and to improve the
     efficacy of scientific investigation. This package provides functions to
-    link the Community Land Model, versions 4.5 and 5, to PEcAn.
+    link the Community Terrestrial System Model, versions 5, to PEcAn. Will 
```

Comment:
one thing we really need to do is figure out how to be consistent about naming ... is CTSM a synonym for CLM5?  which should we use? Can we just say that CLM5 is a synonym for CTSM once, and then use CTSM everywhere else?

Reply:
Don't think it's a synonym. In issue #2122 we say that CTSM package is a support package for LM5 and FATES.